### DISCUSSION TITLE
How do I see history of activity on a project board

### DISCUSSION THREAD

**Discussion by simon-swords** [community/community#28025] [Projects and Issues]
I need to be able to create a query or view of a project board that shows me a row by row history of all changes to cards on that board.

For example,

Fix bug ABC moved from Backlog to Ready by Daniel on Tuesday @ 09:45
Fix bug XYZ moved from Ready to In Progress by Ian on Tuesday @ 09:00
Card created for bug SDF in Ready by Simon on Tuesday @ 08:30
etc

**Comment by gouku** [comment]
It should definitely provide some ways to track what's happening for an issue/item/card. At the moment there's no way to know who and when moves the card or other events like iteration updates. These prevent us from migrating from other tools like ZenHub.

And I think https://github.com/community/community/discussions/8950 is also the same feature request like this.

**Comment by mtgorton-um** [comment]
I agree. It is important to see all activity on a card -- where it came from and where it was moved to (status aka column), as well as who moved it and when. Example: xName moved this card from xStatus/xColumn to xStatus/xColumn, xDate.
